sql/core/src/test/resources/sql-tests/inputs/datetime.sql

```diff
@@ -138,25 +138,11 @@ select to_timestamp("2019 40", "yyyy mm");
 select to_timestamp("2019 10:10:10", "yyyy hh:mm:ss");
 
 -- Unsupported narrow text style
-select date_format(date '2020-05-23', 'GGGGG');
```

Comment:
why we remove these tests?

Reply:
`datetime-formatting-invalid.sql` should be enough to cover these cases

Reply:
so it's not related to this PR? maybe better to open a new PR to remove duplicated test cases.